src/app/components/blog/PostContent.tsx

```
@@ -0,0 +1,45 @@
+          el.innerHTML = katex.renderToString(el.textContent, {
+            displayMode: true,
+          });
+        }
+      });
+    }
+  }, [content]);
+
+  return (
+    <div
+      className="prose-content"
+      dangerouslySetInnerHTML={{ __html: content }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)